code/game/objects/items/handcuffs.dm

```diff
@@ -402,6 +406,12 @@
 		qdel(src)
 	. = ..()
 
+/obj/item/restraints/legcuffs/bola/energy/emp_act(severity)
+	if(prob(25 * severity))
+		return
```

Comment:
higher severity the lesser chance to destroy it?
imo this shouldn't be subject to rng at all

Reply:
EMP severity works by higher = less severe.